src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/ApprovedPremisesRoomsSeedFromXLSXJob.kt

```diff
@@ -0,0 +1,166 @@
+//    "Does this AP have bathroom facilities that have been adapted for wheelchair users?" to "hasWheelChairAccessibleBathrooms",
+//    "Is there a lift at this AP?" to "hasLift",
+//    "Does this AP have tactile & directional flooring?" to "hasTactileFlooring",
+//    "Does this AP have signs in braille?" to "hasBrailleSignage",
+//    "Does this AP have a hearing loop?" to "hasHearingLoop",
+//    "Are there any additional restrictions on people that this AP can accommodate?" to "additionalRestrictions",
+//    "Is the room using only furnishings and bedding supplied by FM?" to "isFullyFm",
+//    "Does this room have Crib7 rated bedding?" to "hasCrib7Bedding",
+//    "Is there a smoke/heat detector in the room?" to "hasSmokeDetector",
+//    "Is this room on the top floor with at least one external wall and not located directly next to a fire exit or a protected stairway?" to "isTopFloorVulnerable",
+//    "Is the room close to the admin/staff office on the ground floor with at least one external wall and not located directly next to a fire exit or a protected stairway?" to "isGroundFloorNrOffice",
+//    "is there a water mist extinguisher in close proximity to this room?" to "hasNearbySprinkler",
+//    "Is this room suitable for people who pose an arson risk? (Must answer yes to Q; 6 & 7, and 9 or  10)" to "isArsonSuitable",
+//    "Is this room currently a designated arson room?" to "isArsonDesignated",
+//    "If IAP - Is there any insurance conditions that prevent a person with arson convictions being placed?" to "hasArsonInsuranceConditions",
+//    "Is this room suitable for people convicted of sexual offences?" to "isSuitedForSexOffenders",
+//    "Does this room have en-suite bathroom facilities?" to "hasEnSuite",
+//    "Are corridors leading to this room of sufficient width to accommodate a wheelchair? (at least 1.2m wide)" to "isWheelchairAccessible",
+//    "Is the door to this room at least 900mm wide?" to "hasWideDoor",
+//    "Is there step free access to this room and in corridors leading to this room?" to "hasStepFreeAccess",
+//    "Are there fixed mobility aids in this room?" to "hasFixedMobilityAids",
+//    "Does this room have at least a 1500mmx1500mm turning space?" to "hasTurningSpace",
+//    "Is there provision for people to call for assistance from this room?" to "hasCallForAssistance",
+//    "Can this room be designated as suitable for wheelchair users?   Must answer yes to Q23-26 on previous sheet and Q17-21 on this sheet)" to "isWheelchairDesignated",
+//    "Can this room be designated as suitable for people requiring step free access? (Must answer yes to Q23 and 25 on previous sheet and Q19 on this sheet)" to "isStepFreeDesignated",
+    "Is this room located on the ground floor?" to "IsGroundFloor",
+  )
+
+  val questionToCharacterEntityMapping = questionToPropertyNameMapping.map { (key, value) ->
```

Comment:
I think this should be probably be a function so we resolve the characteristics everytime they are required (e.g. get hold of this list once in the seed job when processing the excel). These things rarely change but they are changed directly in the DB this list will be stale

Reply:
Raised ticket [APS-1734](https://dsdmoj.atlassian.net/browse/APS-1734) to do this.